packaging/windows/mezmo-agent.wxs

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <?define bitness = "(64 bit)" ?>
+    <?define Win64 = "yes" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFiles64Folder" ?>
+    <Product Id="*" Name="Mezmo Agent $(var.bitness)" Language="1033" Version="$(env.AGENT_VERSION)" Manufacturer="Mezmo" UpgradeCode="a96e8dad-9abd-43e8-a467-307145ecc1e8">
+        <Package InstallerVersion="200" Compressed="yes" InstallScope="perMachine" />
+        <MajorUpgrade DowngradeErrorMessage="A newer version of [ProductName] is already installed." />
+        <MediaTemplate EmbedCab="yes" />
+        <UIRef Id="WixUI_Minimal" />
+        <Feature Id="ProductFeature" Title="Mezmo Agent" Level="1">
+            <ComponentGroupRef Id="ProductComponents" />
+            <ComponentGroupRef Id="ProductComponents2" />
+        </Feature>
+
+        <InstallExecuteSequence>
+            <Custom Action="ServiceRecovery" After="InstallServices">NOT REMOVE</Custom>
+        </InstallExecuteSequence>
+        <Property Id="cmd" Value="cmd.exe" />
+        <CustomAction Id="ServiceRecovery"
+                      ExeCommand="/c start /MIN sc failure MezmoAgentService reset= 60 actions= restart/15000/restart/15000/restart/15000"
+                      Execute="deferred"
+                      Property="cmd"
+                      Impersonate="no"
+                      Return="check" />
+
+    </Product>
+    <Fragment>
+        <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="ProgramFiles64Folder">
+                <Directory Id="INSTALLFOLDER" Name="Mezmo" />
+            </Directory>
+            <Directory Id="CommonAppDataFolder">
+                <Directory Id="DATAFOLDER" Name="logdna" />
+            </Directory>
+        </Directory>
+    </Fragment>
+    <Fragment>
+        <ComponentGroup Id="ProductComponents" Directory="INSTALLFOLDER">
+            <Component Id="ProductComponent" Win64="$(var.Win64)" Guid="56CD2588-B976-4198-B815-FAB7E1E57CD7">
+                <File Id="AgentSvc" Name="logdna-agent-svc.exe" DiskId="1" Source="$(env.AGENT_SVC_EXE_PATH)" KeyPath="yes" />
```

Comment:
I think we might run into upgrade issues if we're not specifying the FileVersion, afaik cargo doesn't set windows resource info in binaries by default, so the MSI will never have any way to infer get ProductVersion unless we provide it or compile it into the agent

Reply:
Current MSI is configured in such way that new build will overwrite old files. 
Your Windows resources PR will be good for user to see version and icon of exe. Currently version and icon are visible in Add/Remove Progs.